runner/pools.go

```diff
@@ -99,12 +99,8 @@ func (r *Runner) UpdatePoolByID(ctx context.Context, poolID string, param params
 		return params.Pool{}, runnerErrors.NewBadRequestError("min_idle_runners cannot be larger than max_runners")
 	}
 
-	if param.Tags != nil && len(param.Tags) > 0 {
-		newTags, err := r.processTags(string(pool.OSArch), pool.OSType, param.Tags)
-		if err != nil {
-			return params.Pool{}, errors.Wrap(err, "processing tags")
-		}
-		param.Tags = newTags
+	if len(param.Tags) == 0 {
```

Comment:
It's not mandatory to send labels as part of the update request. Sending labels is only mandatory during create operations.

Given that we no longer have to append the default labels to supplied labels (if any were supplied), we can just remove any check made against `param.Tags`

Reply:
done